packages/galpy/meta.yaml

```diff
@@ -27,10 +27,6 @@ requirements:
 test:
   imports:
     - galpy
-    - galpy.potential
```

Comment:
Put these back and add galpy to the list of `UNSUPPORTED_PACKAGES` for `node` on line 25 of `packages/test_packages_common.py`